cdap-ui/app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.js

```diff
@@ -50,6 +51,17 @@ export function changeNamespace(namespace) {
   });
 }
 
+function setConfigureError(err) {
+  console.log('Error configuring trigger', err);
```

Comment:
nit: Do we still need this message?

Reply:
just for posterity, since we are attempting to extract error message, just in case there are extra information